File: WebContent/org/r3/db/mongo/timelinecollection/TimelineCollectionDTO.js
//
// Understanding Department - 2016 
//
//
// Mongo - TODO: Readme Section.
//


jQuery.r3Utils.namespace("org.r3.db.mongo.timelinecollection");

org.r3.db.mongo.timelinecollection.TimelineCollectionDTO = Class.extend(
{
	 init: function(aDataBean)
	 {  
       this.dtrowclass   = "grade_timelinecollection";
       this.dtokey       = aDataBean.dtokey   || null;
       this.dtodescr     = aDataBean.dtodescr || null;
       
       this.timelineid = aDataBean.timelineid || null;
       this.site = aDataBean.site || null;
       this.title = aDataBean.title || null;
       this.description = aDataBean.description || null;
       this.timelinetype = aDataBean.timelinetype || null;
       this.creationdate = aDataBean.creationdate || null;
       this.lastmodified = aDataBean.lastmodified || null;
       this.allowedcontent = aDataBean.allowedcontent || null;
	 }
       
});

File: src/org/r3/db/mongo/timelinecollection/TimelineCollectionDTO.java
//
// Understanding Department - 2016 
//
//
// Mongo - TODO: Readme Section.
//

package org.r3.db.mongo.timelinecollection;

import org.mongodb.morphia.annotations.Entity;
import org.r3.db.MongoDbSchemaInfo;

/*
 * Please Define in MongoDbSchemaInfo: public static final String MongoDbTimelineCollectionName = "r3_timelinecollection";
 */
@Entity(value = MongoDbSchemaInfo.MongoDbTimelineCollectionName, noClassnameStored = true)
public class TimelineCollectionDTO extends TimelineCollectionDTOBase
{
	public TimelineCollectionDTO()
	{
		super();
	}

	public java.lang.String getDtokey()
	{
		return timelineid.toString();
	}

	public java.lang.String getDtodescr()
	{
		return title;
	}
}
